webpack/move_to_foreman/components/common/table/components/CollapseSubscriptionGroupButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+const CollapseSubscriptionGroupButton = ({
+  collapsed, visible, onClick, ...props
+}) => {
+  const iconName = collapsed ? 'angle-down' : 'angle-down';
+  const iconStyle = { visibility: visible ? 'visible' : 'hidden' };
```

Comment:
coudn't we use `classnames` instead of inline styling ?

Reply:
Because it's only about the visibility, I didn't want you to go to a css file to understand what it does.